config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
   root to: 'start_page#show'
+
+  get 'personal_details', to: 'personal_details#new'
```

Comment:
I've got a weak preference for building these routes through `:resources` in the fullness of time, but I don't mind whilst the routes file is still lil'.

Reply:
We're in agreement on this